services/src/main/java/com/chatbot/services/AsyncService.java

```diff
@@ -0,0 +1,151 @@
+package com.chatbot.services;
```

Comment:
Similar to suggestion given by Yogesh for Controller, create an abstract service as well and hangouts specific  service child will have an implementation